CCSocialShare/android/java/src/org/kawaz/socialshare/SocialShare.java

```diff
@@ -105,4 +108,25 @@ static public void onActivityResult(int requestCode, int resultCode, Intent inte
         }
         executeCallback(PostResult.SUCCEED.ordinal());
     }
+
+    static public boolean isTwitterAvailable () {
```

Comment:
spaces before `()` is not needed.

`static public boolean isTwitterAvailable()`